docs/source/roadmap.md

```diff
@@ -0,0 +1,268 @@
+4. API for processing multiple FOVs for each SpaceTx group given data in SpaceTx format and a pipeline recipe
+5. Translate each example pipeline into an example pipeline recipe
+6. Visualization tooling to enable parameter selection on small datasets to tweak pipeline recipes
+7. API leverages local parallelism
+
+### A way to run _starfish_ at scale to process SpaceTx datasets
+1. _starfish_-based solution to process multiple FOVs at scale tied to a specific infrastructure (Not designed to be run by SpaceTx users on their hardware) (0/9)<sup>[4](#fn4)</sup>
+
+### Documentation such that new users can learn to use it without our involvement
+1. Public and Private API is fully typed and documented
+2. Purpose of each pipeline component is described, and guidance on parameter tuning, if necessary, is provided
+3. There are examples of formatting data in SpaceTx format
+4. There are examples of fully-worked pipelines
+5. There are examples of the workflow for creating new pipelines
+7. Public API is stable
+
+### _starfish_ facilitates easy interaction with external visualization software (Napari, FIJI) to enable users to tune image processing parameters
+1. _starfish_ has a method to visualize an ImageStack in Napari
+2. _starfish_ has a method to plot spots from an IntensityTable onto an ImageStack in Napari
+3. _starfish_ can dump ImageStacks for viewing 2D TIFFs in FIJI
+
+### Experiment-scale output formats can be loaded on local computers and are adequately expressive to enable QC workflows specified by our users. These output formats enable scientific investigation of the data.
+1. _starfish_ is used to process the three example datasets that we have access to:
+    1. ISS Breast cancer. 16 FOVs with shape (4, 4, 1, 1024, 1024),
+    2. MERFISH U2-OS cells. 400 FOVs with shape (8, 2, 1, 2048, 2048),
+    3. osmFISH Visual Cortex. ~100s FOVs with shape (13, 3, 45, 2048, 2048)
+2. Outputs from Fields of view can be combined into a single output format
+3. Output formats can be loaded and are performant
+4. Processed data can be used to recapitulate results
+
+### _starfish_ output formats enable external annotation to place the data in anatomical and cell type context and allow comparison of across methods
+1. Processed Experiments (all FOVs for a given dataset) can produce Cell x Gene tables with each cell in the physical coordinates of the sample
+2. 2D polygon annotations of anatomical structures in physical coordinates can be saved in a standardized format and membership of each cell in annotation structures can be added as metadata to the Cell x Gene table
+3. Cell x Gene table can be easily imported by computational biologists (outside of _starfish_) to determine cell type or cluster membership for each cell.  This cluster membership can include probabilistic membership to multiple clusters and is added to the Cell x Gene table as metadata
+
+### A tool to convert image data into SpaceTx Format
```

Comment:
@joshmoore leaving you a TODO to translate OME terminology to SpaceTx when you have a translation you're happy with.